### PR TITLE
fix: link menu overflow in right

### LINKED ISSUE #375: [Bug] Overflow in insert link text field while not in full screen mode
### Bug Description

There is text field overflow while inserting a link if the editor is not in full-screen mode.

Current Behavior:
9999fb75660cd189e4056776db5df0c1bbb4b55b

Expected Behavior:
9627252d7e5d5b7936c97a5964509ebc16fca2e5

### How to Reproduce

--

### Expected Behavior

--

### Operating System

--

### AppFlowy Editor Version(s)

--

### Screenshots

_No response_

### Additional Context

Reported on the forum by Yatendra

https://forum.appflowy.io/t/overflow-in-insert-link-text-field-while-not-in-full-screen-mode/90

### PR DESCRIPTION
**Pull request by sun-jiao**
Close #375 .

### CODE REVIEW

File: lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart [hunk 1]
```diff
@@ -120,3 +109,54 @@ void showLinkMenu(
 
   Overlay.of(context).insert(overlay!);
 }
+
+(double? left, double? top, double? right, double? bottom) getPosition(
```
Comment (LucasXu0) [inline lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart:113]: mark as a private function.

File: lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart [hunk 2]
```diff
@@ -120,3 +109,54 @@ void showLinkMenu(
...
+    editorOffset.dy,
+    linkText != null ? _hasTextHeight : _noTextHeight,
+    rect.top,
+    rect.bottom,
+  );
+
+  return (left, top, right, bottom);
+}
+
+(double? start, double? end) getLength(
```
Comment (LucasXu0) [inline lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart:145]: - Add type to all parameters.
- Add comments to describe its functionality.
- Mark as private.

File: lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart (hunk 1 above)
Comment (sun-jiao) [reply lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart]: Done.

File: lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart (hunk 2 above)
Comment (sun-jiao) [reply lib/src/editor/toolbar/desktop/items/link/link_toolbar_item.dart]: Done

**Comment by sun-jiao** [comment]
So many codecov errors recently.

**Pull request merged**